ci/da-ghc-lib/compile.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   variables:
     ghc-lib-sha: '42e5c306dcfbc84b83336fdd531023e93bfcc5b2'
     base-sha: '9c787d4d24f2b515934c8503ee2bbd7cfac4da20'
-    patches: 'f369e0f5c6933e895f90ae4efcb3d4294aba532b 833ca63be2ab14871874ccb6974921e8952802e9'
+    patches: 'a2d600c659760dccbdcfb9c10c6a7138ef0c23be 833ca63be2ab14871874ccb6974921e8952802e9'
```

Comment:
can you open a PR for that as well so we can review the changes?

Reply:
https://github.com/digital-asset/ghc/pull/72